### PR TITLE
Fix PARENT_IMG error on pull requests

### PR DESCRIPTION
**Pull request by avisiedo**
- Read PARENT_IMG from the repository for the pipeline.
- Default value on Makefile from the repository and allow to overrides it.

### CODE REVIEW

File: .github/workflows/containers.yaml [hunk 1]
```diff
@@ -53,14 +54,15 @@ jobs:
...
           dockerfiles: |
             ./Dockerfile
           build-args: |
-            PARENT_IMG=${{ secrets.PARENT_IMG }}
-            QUAY_EXPIRATION=${{ steps.getTags.outputs.QUAY_EXPIRATION }}
+            PARENT_IMG=${{ steps.getParams.outputs.PARENT_IMG }}
+            QUAY_EXPIRATION=${{ steps.getParams.outputs.QUAY_EXPIRATION }}
 
       - name: Push To quay.io
+        if: ${{ github.event_name != 'pull_request' }}
```
Comment (tiran) [inline .github/workflows/containers.yaml:65]: ``if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true``

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 include private.mk
 
-PARENT_IMG ?= quay.io/freeipa/freeipa-server:fedora-34
+PARENT_IMG ?= $(shell grep -v "^#" PARENT_IMG)
```
Comment (tiran) [inline Makefile:3]: This code is confusing. It took me a while to realize that you have also a file called ``PARENT_IMAGE``.

Comment (tiran) [reply Makefile:3]: maybe rename to ``parent_img.conf`` ?

Comment (avisiedo) [reply Makefile:3]: Thanks! I will rename the file PARENT_IMG to BASE_IMAGE and I will add some comment before the line for clarification.

File: .github/workflows/containers.yaml [hunk 3]
```diff
@@ -30,9 +30,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Get tags
-        id: getTags
+      - name: Get Parameters
+        id: getParams
         run: |
+          echo ::set-output name=PARENT_IMG::$(grep -v "^#" PARENT_IMG)
```
Comment (tiran) [inline .github/workflows/containers.yaml:36]:
```suggestion
          echo "PARENT_IMG=$(grep -v "^#" parent_img.cnf)" >> $GITHUB_ENV
```

File: Makefile (hunk 2 above)
Comment (tiran) [reply Makefile:3]: * add ``ci/config/env`` with shell includes. Comments and empty lines are automatically ignored
* Add ``PARENT_IMG=quay.io/freeipa/freeipa-server@sha256:ba7a482d59a0879bf427ebf8a870cc7f06e6f680c056995933fcbe48b19db0fb`` to the env files
* ``PARENT_IMG ?= $(shell . ci/config/env && echo $PARENT_IMG)``

Comment (avisiedo) [reply Makefile]: Thanks! updated!

File: .github/workflows/containers.yaml (hunk 1 above)
Comment (avisiedo) [reply .github/workflows/containers.yaml]: Secrets are not accessible on pull_request event. I will restore the initial filter. Thanks for the suggestion.

**Pull request merged**